modules/end-user-guide/pages/enabling-artifact-repositories-in-a-restricted-environment.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+:navtitle: Enabling artifact repositories in a restricted environment
+:description: Enabling artifact repositories in a restricted environment
+:keywords: artifact-repositories, artifact-repository, maven, gradle, nuget, python, go, npm
+:page-aliases: .:using-artifact-repositories-in-a-restricted-environment
+
+[id="enabling-artifact-repositories-in-a-restricted-environment_{context}"]
+= Enabling artifact repositories in a restricted environment
+
+By configuring technology stacks, you can work with artifacts from in-house repositories using self-signed certificates:
```

Comment:
(...) using the following self-signed certificates

this would make the sentence flow better imho